observation-service/src/main/java/gov/cdc/etldatapipeline/observation/util/ProcessObservationDataUtil.java

```diff
@@ -405,6 +410,9 @@ private void transformObservationReasons(String observationReasons) {
     }
 
     private void transformObservationTxt(String observationTxt) {
+        // Tombstone message to delete previous observation coded data for specified uid
+        sendToKafka(observationKey, null, txtTopicName, observationKey.getObservationUid(), null);
```

Comment:
👍

Reply:
The comment can be updated for txt. It's got the same comment as observation coded.